docs/release_notes/v1.9.2.md

```diff
@@ -0,0 +1,22 @@
+# Dapr 1.9.2
+
+### Fixes panics when using pubsub subscriptions or input bindings via gRPC with tracing disabled
+
+#### Problem
+
+Users who disabled tracing and are subscribing to Dapr pubsub components via gRPC or using input bindings via gRPC will encounter panics when an event is attempted to be delivered.
+
+### Impact
+
+This issue impacts users who:
+
+- Disabled tracing (sampling rate is set to 0, not the default Dapr config sets this to 1)
```

Comment:
`- Disabled tracing by setting samplingRate to "0" in Dapr's configuration (by default, the value is "1")`

Reply:
```suggestion
- Disabled tracing (sampling rate is set to 0, not the default Dapr config sets this to 1)
```